winpty/src/windows.rs

```diff
@@ -95,12 +111,12 @@ impl Config {
     /// Set the mouse mode
     pub fn set_mouse_mode(&mut self, mode: &MouseMode) {
         let m = match mode {
-            MouseMode::None => 0,
-            MouseMode::Auto => 1,
-            MouseMode::Force => 2,
+            MouseMode::None => WINPTY_MOUSE_MODE_NONE,
+            MouseMode::Auto => WINPTY_MOUSE_MODE_AUTO,
+            MouseMode::Force => WINPTY_MOUSE_MODE_FORCE,
```

Comment:
Always nice to get rid of some magic numbers, thank you!

Reply:
:D indeed, I saw the constants existed and felt I had to use them!